### PR TITLE
forget-password

### PR DESCRIPTION
**Pull request by yashkapoor8710**
Screenshot 2020-04-28 at 7 13 23 PM

Screenshot 2020-04-28 at 7 13 37 PM

Screenshot 2020-04-28 at 7 13 51 PM

### CODE REVIEW

**Comment by yashkapoor8710** [comment]
Screenshot 2020-04-28 at 7 43 49 PM

Screenshot 2020-04-28 at 7 44 09 PM

I have fixed the iphone 8 issue removed the scroll

File: pages/ForgetPassword/index.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  const theme: AppTheme = useTheme();
+
+  const backButton = () => {
+    history.push('/')
+  }
+
+  return (
+    <View style={{flex: 1, flexDirection:'column'}}>
+      <ScrollView>
+        <ImageBackground source={ImagePath} style={{ width: '100%', height: 550,}} >
```
Comment (manojsinghnegiwd) [inline pages/ForgetPassword/index.tsx:38]: I see a lot of inline style not sure why we can't put them below in the StyleSheet object.

Comment (yashkapoor8710) [reply pages/ForgetPassword/index.tsx:38]: I have fixed this

File: pages/ForgetPassword/index.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+              <ThemedText styleKey="highlightTextColor" style={style.textStyle}>{constants.backText}</ThemedText>
+            </View>
+          </TouchableOpacity>
+          <View style={[style.topContainer, {marginTop: 120, marginBottom: 10}]}>
+            <View style={[style.forgetContainer, {backgroundColor: theme.backgroundColor}]}>
+              <Image source={forget} style={[style.logoImage, {width: 50, height: 50}]}/>
+            </View>
+          </View>
+          <View style={[style.topContainer, {marginTop: 0, marginBottom: 30}]}>
+            <ThemedText styleKey="highlightTextColor" style={[style.textStyle, {fontSize: 32, textTransform: 'capitalize'}]}>{constants.forgetText}</ThemedText>
```
Comment (manojsinghnegiwd) [inline pages/ForgetPassword/index.tsx:53]: There is still inline style here

File: pages/ForgetPassword/index.tsx [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+        <ImageBackground source={ImagePath} style={{ width: '100%', height: 550,}} >
+          <TouchableOpacity style={style.backContainer} onPress={backButton}>
+            <View style={style.leftContainer}>
+              <MaterialIcon name="chevron-left-circle-outline" size={30} color={theme.highlightTextColor} style={style.backIcon}/>
+            </View>
+            <View style={style.rightContainer}>
+              <ThemedText styleKey="highlightTextColor" style={style.textStyle}>{constants.backText}</ThemedText>
+            </View>
+          </TouchableOpacity>
+          <View style={[style.topContainer, {marginTop: 120, marginBottom: 10}]}>
```
Comment (manojsinghnegiwd) [inline pages/ForgetPassword/index.tsx:47]: There is still inline style here

File: pages/ForgetPassword/index.tsx [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+            <View style={[style.forgetContainer, {backgroundColor: theme.backgroundColor}]}>
+              <Image source={forget} style={[style.logoImage, {width: 50, height: 50}]}/>
+            </View>
+          </View>
+          <View style={[style.topContainer, {marginTop: 0, marginBottom: 30}]}>
+            <ThemedText styleKey="highlightTextColor" style={[style.textStyle, {fontSize: 32, textTransform: 'capitalize'}]}>{constants.forgetText}</ThemedText>
+          </View>
+          <RoundButton buttonStyle={{minWidth: 230, marginTop: 0}} label={constants.choiceOne} buttonColor={theme.forgetColor} labelStyle={theme.highlightTextColor} />
```
Comment (manojsinghnegiwd) [inline pages/ForgetPassword/index.tsx:55]: There is still inline style here

File: pages/ForgetPassword/index.tsx [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+          <RoundButton buttonStyle={{minWidth: 230, marginTop: 10}} label={constants.resetPass} buttonColor={theme.backgroundColor} labelStyle={theme.appColor} />
```
Comment (manojsinghnegiwd) [inline pages/ForgetPassword/index.tsx:56]: There is still inline style here

File: pages/ForgetPassword/index.tsx (hunk 4 above)
Comment (yashkapoor8710) [reply pages/ForgetPassword/index.tsx]: I have removed inline css

File: pages/ForgetPassword/index.tsx (hunk 5 above)
Comment (yashkapoor8710) [reply pages/ForgetPassword/index.tsx]: I have removed inline css

File: pages/ForgetPassword/index.tsx (hunk 3 above)
Comment (yashkapoor8710) [reply pages/ForgetPassword/index.tsx]: I have removed the inline css

File: pages/ForgetPassword/index.tsx (hunk 2 above)
Comment (yashkapoor8710) [reply pages/ForgetPassword/index.tsx]: I have removed inline css

File: pages/Login/index.tsx [hunk 6]
```diff
@@ -136,7 +140,9 @@ const Login: React.FunctionComponent<Props> = ({
           <ErrorText
             errors={errors.password}
           /> 
-          <ThemedText style={[style.forgotPassword, {fontWeight: 'bold', textAlign: 'right', alignSelf: 'flex-end'}]} styleKey="appColor">{constants.labelForget}</ThemedText>
+          <TouchableOpacity onPress={goToForget}>
+            <ThemedText style={[style.forgotPassword, {fontWeight: 'bold', textAlign: 'right', alignSelf: 'flex-end'}]} styleKey="appColor">{constants.labelForget}</ThemedText>
```
Comment (manojsinghnegiwd) [inline pages/Login/index.tsx:144]: Inline style

Comment (yashkapoor8710) [reply pages/Login/index.tsx]: I have removed the inline css

**Pull request merged**